extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -44,7 +46,7 @@ define('ext.wikia.adEngine.adContext', [
 		context.slots = context.slots || {};
 		context.targeting = context.targeting || {};
 		context.providers = context.providers || {};
-		context.forceProviders = context.forceProviders || {};
+		context.forcedAdProvider = qs.getVal('forcead', null) || context.forcedAdProvider || null;
```

Comment:
or `s/forcedprovider/forcedadprovider/` in query param - although, it makes sense to keep them short and convenient to use, I guess ;)